scripts/build.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import _aqt
+import anki
+import subprocess
+
+
+def vendor_requirements():
+    """Install requirements into the src/media_import/libs directory."""
+    subprocess.run(
+        [
+            "pip",
+            "install",
+            "--no-deps",
+            "--target=./src/media_import/libs",
+            "-r",
+            "requirements.txt",
+        ],
+        check=True,
```

Comment:
https://snarky.ca/why-you-should-use-python-m-pip/

```suggestion
        [
            "python",
            "-m",
            "pip",
            "install",
            "--no-deps",
            "--target=./src/media_import/libs",
            "-r",
            "requirements.txt",
        ],
        check=True,
```